### ISSUE TITLE
As a product owner and information researcher, I need to log user interactions with the Media Suite, so I can monitor its use and plan strategically

### ISSUE THREAD

**Issue by lilimelgar** [CLARIAH/wp5_mediasuite#242]
Product owners and information/computer scientists in the CLARIAH team, as well as trainers/teacher using the Media Suite in workshops, would like to know how many people use it, which parts of the MS are mostly used,  and other analytics to facilitate user studies, information research, and further development.

**Comment by lilimelgar** [comment]
@roelandordelman Shall we close this one if it is a duplicate of #312?

**Comment by lilimelgar** [comment]
@roelandordelman I got notification that this issue is closed, I have a question please just to make sure we can use this solution also in future user evaluations/studies, or if I should open another issue for this: Is it possible to cross information from the analytics system implemented and SurfConnex? or better: is it possible to know this at this moment:
-See amount of use per type of user according to level (e.g., student, assistant professor, professor...)
-See amount of use per type of user according to subject domain - department (e.g., history, media studies...)
-See parts of the MS used by type of user (visited parts + time of stay in each part)

There was also the idea of using OrchidID for identification, is it possible to connect this information with the current solution?

**Comment by roelandordelman** [comment]
@lilimelgar 

It is a CLARIAH wide desire to have the information you mention including ORCID available for authorization (and provenance) purposes, related to SURFConext (and its flavors). The logging of actions is detached from this.  Maybe at some point a joint is possible, but not soon (if ever).

**Comment by lilimelgar** [comment]
@roelandordelman Thanks. Still, for evaluation and improvement, at least of the MS, this information is important. When we had OneGini, we had a form to collect this information, people had to enter it manually for registration. With Themis, we had thought of adding the "Terms of service" document, plus three questions about subject domain, expertise/years in the field, and position. This was not done since we got SurfConnex in the way and the proper solution was to do it as you described above. Thus, I added a new issue (#341), so we keep in mind to evaluate this "manual" option and then decide if placing it in the workflow or close it if the decision is not to do it.
